syllabus/tools.md

```diff
@@ -96,7 +96,7 @@ I use [atom](https://atom.io/), but I decided to use it by downloading both Atom
 
 Video install instructions for Anaconda:
 - [Windows](https://www.youtube.com/watch?v=xxQ0mzZ8UvA)
-- [Mac](https://www.youtube.com/watch?v=TcSAln46u9U)
+- [Mac](https://www.youtube.com/watch?v=TcSAln46u9U) (link for if you'd like to install python via environment: https://opensource.com/article/19/5/python-3-default-mac )
```

Comment:
```suggestion
- [Mac](https://www.youtube.com/watch?v=TcSAln46u9U) 

On Mac,  to install python via environment, [this article may be helpful](https://opensource.com/article/19/5/python-3-default-mac)
```

I broke up your contribution and made it into a markdown link, so it will render properly.  I'll let you commit this, so that you can see how it works, and then I'll merge.